burnman/eos/hp.py

```diff
@@ -272,7 +277,22 @@ def __intCpoverTdT (self, temperature, params):
         """
         return (params['Cp'][0]*np.log(temperature) + params['Cp'][1]*temperature - 0.5*params['Cp'][2]/np.power(temperature,2.) - 2.0*params['Cp'][3]/np.sqrt(temperature)) - (params['Cp'][0]*np.log(T_0) + params['Cp'][1]*T_0 - 0.5*params['Cp'][2]/(T_0*T_0) - 2.0*params['Cp'][3]/np.sqrt(T_0))
 
+    def _magnetic_gibbs(self, pressure, temperature, params):
+        """
+        Returns the magnetic contribution to the Gibbs free energy
```

Comment:
Holy guacamole, these equations baffle me. How do people get to these ratios? As far as I can follow it, it looks good. Could you add the Sundman reference here?